production/queries/select_course.sql

```diff
@@ -1,29 +1,25 @@
 SELECT
```

Comment:
The select course query now reads from a master materialized view of course details (with offerings for the current year) rather than performing left joins across multiple tables.